GitCommands/Config/SettingKeyString.cs

```diff
@@ -65,5 +65,31 @@ public static class SettingKeyString
         /// protocol.file.allow
         /// </summary>
         public static string AllowFileProtocol = "protocol.file.allow";
+
+        /// <summary>
+        /// commit.gpgSign  Determines if commits are signed by default
+        /// </summary>
+        /// <remarks>
+        /// Min version 2.0
+        /// </remarks>
+        public static string CommitGPGSign = "commit.gpgSign";
```

Comment:
```suggestion
        public static string CommitGpgSign = "commit.gpgSign";
```

Ditto everywhere, including type names.